app/controllers/admin/facilities_controller.rb

```diff
@@ -8,16 +8,42 @@ class Admin::FacilitiesController < AdminController
   before_action :initialize_upload, :validate_file_type, :validate_file_size, :parse_file,
     :validate_facility_rows, if: :file_exists?, only: [:upload]
 
+  skip_after_action :verify_authorized, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  skip_after_action :verify_policy_scoped, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  after_action :verify_access_authorized, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+
   def index
-    authorize([:manage, :facility, Facility])
+    if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+      current_admin.authorize(:manage, :facility)
+    else
+      authorize([:manage, :facility, Facility])
+    end
```

Comment:
```
if Flipper.enabled?
     check new permissions
else
     check old permissions 
end
```

This pattern gets repeated throughout. We can get rid of all these conditionals when we make the full transition